src/components/form-field/radio-box/radio-box-option.tsx

```diff
@@ -0,0 +1,73 @@
+import { RadioGroup } from "@headlessui/react";
+import React from "react";
+import { classNames } from "../../../util/class-names";
+
+export interface RadioBoxOptionProps {
+    children: React.ReactNode;
+    value: string;
+    disabled?: boolean;
+    highlightText?: string;
+}
+
+export const RadioBoxOption = ({
+    children,
+    value,
+    disabled,
+    highlightText,
```

Comment:
This could be a potential subcomponent of the option component. If we need to add an icon or other special design requirements, this would be a bit more flexible. But it is just an idea. It is good as it is and only optional!

Reply:
Sounds really valid to me. I just outsourced it to a new component + storybook file we now can use for other components as well (in theory) ☺️:

![CleanShot 2023-11-14 at 13 18 10@2x](https://github.com/abusix/hailstorm/assets/4981006/f7f66fc4-6000-40b9-adc1-5358e10a222e)